api/qupo_backend/tickers_api.py

```diff
@@ -0,0 +1,105 @@
+import logging
+
+from fastapi import APIRouter, Depends, HTTPException
+from pytickersymbols import PyTickerSymbols
+from sqlalchemy.orm import Session, sessionmaker
+
+from config import settings
+from .db import models, schemas
+from .db.database import SessionLocal, engine
+from .integrator import get_all_symbols, get_data_of_symbol
+
+
+router = APIRouter(
+    prefix='/tickers',
+    tags=['tickers'],
+)
+
+settings = settings()
+stock_data = PyTickerSymbols()
+
+
+def get_db():
+    if(settings['DATABASE']['USE_DB']):
+        models.Base.metadata.create_all(bind=engine)
+        db = SessionLocal()
+        try:
+            yield db
+        finally:
+            db.close()
+    else:
+        yield sessionmaker(bind=None)
+
+
+@router.get('/symbols')
+async def get_symbols(symbols_only=False):
+    try:
+        return get_all_symbols(stock_data, symbols_only)
+    except Exception as e:
```

Comment:
maybe, do we want this pattern to be repeated? what is the default behaviour for fastify? if it logs the error anyway and gives 500, only benefit is the custom 'could not get any ...' message, which might not be required.